### ISSUE TITLE
Compilation problems

### ISSUE THREAD

**Issue by denisvmedia** [a0rtega/pafish-dll#1]
Hi, I tried to compile this library, but it doesn't. It seems it requires pafish files. But what you have in the pafish repo, seems to be incompatible with this lib. Where can I download an old version of pafish? Or, maybe, by any chance, you have a fresher version of this dll? Thanks.

**Comment by a0rtega** [comment]
Hi,

Im not maintaining this DLL anymore, as the users base seems to be limited.

Anyway, if you are interested, all pafish releases are tagged and you can
download the repository on each point. You may try with that to find the
version supported by the DLL.

Kind regards!

**Comment by a0rtega** [comment]
I almost forgot that you have the compiled version in the repository as
well. Look for pafish.dll file.

**Comment by denisvmedia** [comment]
Yes, but I wanted to re-compile it. I tried taking the oldest version of pafish in repo, but it's still younger than dll, and it doesn't compile either. Maybe, by occasion, you have a 2yo release of pafish?
